zephyr/CMakeLists.txt

```diff
@@ -133,6 +133,7 @@ if (CONFIG_SOC_SERIES_INTEL_CAVS_V15)
 
 	zephyr_library_sources_ifdef(CONFIG_INTEL_DMIC
 		${SOF_DRIVERS_PATH}/intel/dmic/dmic.c
+		${SOF_DRIVERS_PATH}/intel/dmic/dmic_computed.c
```

Comment:
There's ssp/ssp.c and support module mn.c. I could rename dmic_computed.c to computed_setup.c and dmic_nhlt.c to blob_setup.c. I'm not good with names, if reviewers like those I can change. Or propose new names. I got somewhere else feedback that I should change "nhlt" to "blob". I don't like myself word blob so much because it's not descriptive, (instead perhaps suggests that you don't want to understand what's there). But also FW  is not accessing BIOS NHLT so also it's not an ideal name for the thing. Please propose!